### DISCUSSION TITLE
Why this project is not participated in hacktoberfest

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#292] [General]
Please involve this project in hacktoberfest show that more and more contributions come and contribute in this project

**Comment by T-Crypt** [comment]
@mahaveergurjar 

Hacktoberfest has strict requirements for their events that are participating. It would end up being more work to get this setup as a Hacktoberfest event then it would be useful. 

Its mostly for Open-Source projects / Open Source Development and this is purely for user configuration / user preference as they are dotfiles and not an Open Source Project.

Check out Zero-To-Mastery if you want to participant in Hacktoberfest
